src/pages/resources/migration/android/tabs/migrate-to-2x.md

```diff
@@ -0,0 +1,49 @@
+<Variant lang="java" repeat="1"/>
+
+```java
+public class MainApp extends Application {
+    private static final String ENVIRONMENT_FILE_ID = "ENVIRONMENT_FILE_ID";
```

Comment:
Should we use `YOUR_ENVIRONMENT_FILE_ID` to indicate you need to replace this value? (same comment for all occurrences